jca-and-hibernate/pom.xml

```diff
@@ -43,7 +43,7 @@
         <version.org.hibernate>4.2.7.SP1</version.org.hibernate>
         <version.org.hibernate.hibernate-validator>5.0.1.Final</version.org.hibernate.hibernate-validator>
         <version.org.jboss.jboss-common-core>2.2.17.GA</version.org.jboss.jboss-common-core>
-        <version.org.jboss.narayana>${project.version}</version.org.jboss.narayana>
+        <version.narayana>${project.version}</version.narayana>
```

Comment:
I assume that it is should be "<version.narayana>5.3.3.Final-SNAPSHOT</version.narayana>